reference/ros-api.md

```diff
@@ -359,6 +359,7 @@ for more information about the value of `robot_unique_name`).
 | [/supervisor/get\_self](supervisor.md#wb_supervisor_node_get_from_def)                                                | service       | webots\_ros::supervisor\_get\_self                                   | uint8 ask<br/>---<br/>uint64 node                                                                                                                                                                                                                                 |
 | [/supervisor/get\_from\_def](supervisor.md#wb_supervisor_node_get_from_def)                                           | service       | webots\_ros::supervisor\_get\_from\_def                              | string name<br/>---<br/>uint64 node                                                                                                                                                                                                                               |
 | [/supervisor/get\_from\_id](supervisor.md#wb_supervisor_node_get_from_def)                                            | service       | webots\_ros::supervisor\_get\_from\_id                               | int32 id<br/>---<br/>uint64 node                                                                                                                                                                                                                                  |
+| [/supervisor/get\_selected](supervisor.md#wb_supervisor_node_get_from_def)                                            | service       | webots\_ros::supervisor\_get\_selected                               | uint8 ask<br/>---<br/>uint64 node                                                                                                                                                                                                                                 |
```

Comment:
I agree, I will rename this service type to `webots_ros::supervisor_get_node` so that it is more generic, and we will be able to use it for other services in the future.